packages/ember-cli-fastboot/package.json

```diff
@@ -46,6 +46,7 @@
     "body-parser": "^1.18.3",
     "broccoli-asset-rev": "^3.0.0",
     "broccoli-test-helper": "^1.5.0",
+    "co": "4.6.0",
```

Comment:
https://github.com/ember-fastboot/ember-cli-fastboot/blob/b6b17af9dfb308919b41b07c610397add3adcf90/packages/ember-cli-fastboot/test/new-package-json-test.js#L8
Looks like `co` isn't actually being used, we should be able to remove